rpcserver.go

```diff
@@ -1113,6 +1113,11 @@ func (r *rpcServer) GetInfo(ctx context.Context,
 		activeChains[i] = chain.String()
 	}
 
+	nodeAnnouncement, err := r.server.genNodeAnnouncement(false)
```

Comment:
Why do we need to generate a new announcement _each_ time? The only thing we require here is the alias, which is already known at this time: either we look at the config, or we load a single announcement into memory upon start up.

Reply:
It looks like calling `genNodeAnnouncement(false)` just returns the current node announcement, but does so with thread-safety (per [this](https://github.com/lightningnetwork/lnd/pull/467#discussion_r155923861) discussion).

I could refactor this to explicitly lock the server mutex and pull the alias from the current announcement without calling `genNodeAnnouncement`, to avoid confusion. Thoughts?

Reply:
Ah, totally missed that. Hadn't realized that method had changed recently :p 

Nah, I think this is fine as is after all